src/common/example/index.ts

```diff
@@ -32,7 +33,7 @@ export class App extends WidgetBase<WidgetProperties> {
 	}
 
 	render() {
-		return v('div', [
+		return v('div', { id: 'module-select' }, [
```

Comment:
The `Toolbar` uses a flexbox-based layout wherein page content should be a child of the toolbar itself. This negates the need for any `padding-top` on the `<body>` element to account for a fixed-positioned toolbar, and allows the toolbar to be an arbitrary height while still allowing page content to scroll independently.

Unfortunately, because our example pages render this module select UI _before_ any component is rendered, it's impossible to place this inside the `Toolbar` example. To get around this, the toolbar example page adds a single CSS rule to the example page to style the select in a more sane manner, hence this added ID.